examples/testify/src/integration-test/groovy/gorm/tools/security/AppUserRepoSpec.groovy

```diff
@@ -44,6 +44,40 @@ class AppUserRepoSpec extends Specification implements DataIntegrationTest, Secu
         passwordEncoder.matches(params.password, user.passwordHash)
     }
 
+    void "test unique"() {
```

Comment:
need to test the tokenController now too and make sure the rest calls will get a sane message back